specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2023-03-01/costmanagement.benefits.json

```diff
@@ -314,6 +314,636 @@
           "nextLinkName": "nextLink"
         }
       }
+    },
+    "/providers/Microsoft.Billing/billingAccounts/{billingAccountId}/providers/Microsoft.CostManagement/generateBenefitUtilizationSummariesReport": {
+      "post": {
+        "tags": [
+          "BenefitUtilizationSummariesAsync"
+        ],
+        "operationId": "GenerateBenefitUtilizationSummariesReportAsync_BillingAccountScope",
+        "description": "Triggers generation of a benefit utilization summaries report for the provided billing account. This API supports only enrollment accounts.",
+        "externalDocs": {
+          "url": "https://docs.microsoft.com/rest/api/cost-management/"
+        },
+        "x-ms-examples": {
+          "GenerateUtilizationSummariesReportByBillingAccount": {
+            "$ref": "./examples/BenefitUtilizationSummaries/Async/GenerateBenefitUtilizationSummariesReportByBillingAccount.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
```

Comment:
>location [](http://example.com/codeflow?start=30&length=8)

if any of these async POST APIs are exposing what could be considered personal data or secrets and you need to release these within the next month or two please implement step 4 here:

[https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/async-api-reference.md#operation-result-rules](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/async-api-reference.md#operation-result-rules)

ARM will be introducing a way to enforce that centrally in the near future but its not ready yet